example-monorepo/packages/plugin-test/.env.schema

```diff
@@ -0,0 +1,27 @@
+# @defaultRequired=infer @defaultSensitive=false
+# @generateTypes(lang=ts, path=env.d.ts)
+#
+# @currentEnv=$APP_ENV
```

Comment:
`@envFlag=APP_ENV` is now deprecated (although still works) in favor of `@currentEnv=$APP_ENV`

Unifying the resolvers should make things more clear

Reply:
yeah makes sense to me, probably good for us to draft some guidelines about these types of deprecations though